documentation/docs/examples/clipping.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: clipping
+title: Clipping/slicing planes
+describe: Reveal support adding a number of slicing planes to "cut through" the models.
+---
+
+import { DemoWrapper } from '@site/src/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Slicing planes, also called clipping planes, can be used to "cut through" the model to reveal the interior. This is 
+useful to isolate interesting parts of the model and hide obscuring geometry. Reveal supports having multiple slicing planes
+in arbitrary position and orientation. When slicing planes are active,
+the amount of geometry loaded might be reduced since culled geometry doesn't need
+to be loaded (or rather loaded in a lower detail level).
+
+<video preload="auto" autoPlay="autoplay" loop controls>
+  <source src={useBaseUrl('/img/cad-clipping.webm')} />
+</video>
+
+:::warning
```

Comment:
There is no warning https://v2.docusaurus.io/docs/markdown-features#calloutsadmonitions

Perhaps it should be:
```suggestion
:::caution
```

Reply:
Interestingly, warning actually works (but caution is better here)